### ISSUE TITLE
Add search when selecting items in loadouts.

### ISSUE THREAD

**Issue by SilverBeechStudios** [Sidetalker/SplatPal#33]
I think this would speed up loadout creation, as well as other areas in the app if search was added to all lists that could benefit from it.

**Comment by Sidetalker** [comment]
Good idea, this would definitely be helpful. I'm guilty of not actually using my own loadout creator for anything but testing, so I sometimes miss out on usability things like this!

**Comment by SilverBeechStudios** [comment]
I forget how it works, but would be nice if the search box was live right as you enter the view, so you can just hit "New Loadout" then type "D" and it takes you to Dynamo.

**Comment by Sidetalker** [comment]
You got it boss
